specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-10-27-preview/types/privateAccesses.json

```diff
@@ -0,0 +1,245 @@
+      "type": "object",
+      "description": "The properties of a private access resource",
+      "properties": {
+        "privateEndpointConnections": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          },
+          "description": "A readonly collection of private endpoint connection. Currently only one endpoint connection is supported."
+        }
+      }
+    },
+    "PrivateAccessListResult": {
+      "type": "object",
+      "description": "Model that represents a list of private access resources and a link for pagination.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateAccess"
+          },
+          "description": "List of private access resources.",
+          "readOnly": true
+        },
+        "nextLink": {
+          "$ref": "./common.json#/definitions/urlNullable",
+          "description": "URL to retrieve the next page of private access resources.",
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateLinkResourceListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Array of private link resources",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          }
+        }
+      },
+      "description": "A list of private link resources"
+    },
+    "PrivateLinkResource": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkResourceProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "private link resource Id"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "private link resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "private link resource type"
+        }
+      },
+      "description": "A private link resource"
+    },
+    "PrivateLinkResourceProperties": {
+      "properties": {
+        "groupId": {
+          "description": "The private link resource group id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "requiredMembers": {
+          "description": "The private link resource required member names.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        },
+        "requiredZoneNames": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The private link resource DNS zone name."
+        }
+      },
+      "description": "Properties of a private link resource."
+    },
+    "PrivateEndpoint": {
```

Comment:
this is also already definedin https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/privatelinks.json

Reply:
these definitions are still here